.storybook/main.js

```diff
@@ -1,23 +1,27 @@
 const path = require("path");
 const glob = require("glob");
-
 const projectRoot = path.resolve(__dirname, "../");
 const ignoreTests = process.env.IGNORE_TESTS === "true";
 const isChromatic = !ignoreTests;
 const getStories = () =>
-  glob.sync(`${projectRoot}/src/**/*.stories.@(js|mdx|tsx)`, {
+  glob.sync(`${projectRoot}/src/**/*.@(mdx)`, {
+    ...(ignoreTests && {
+      ignore: `${projectRoot}/src/**/*.stories.@(js|mdx|tsx)`,
```

Comment:
This is just basically what was already there. The only changes are that stories are only `*.mdx` now and there is no `*.stories.*` as part of a change Storybook have made between SB6 and SB7. And to simplify it I must have removed the `test` specific part from the `ignore:` as we don't have any `.stories` anymore except for the Test Stories. I could put it back to have the `-test.` part again but I'm not sure it's going to make much difference per se?

Reply:
As for the getCSFStories() that seems a bit of an oversight, so I will look into all of this properly just to confirm

Reply:
I think I'm mostly just a bit confused, on the whole I'm happy to leave this as it is if you're happy it's doing what it should be in all situations.

So looking at it again, I do think it's fine because `glob.sync` just returns an array of filenames, and we also have the `getCSFStories`, which isn't ignoring these non-test stories and therefore will add these to the list of files we're importing. In fact I guess the ignoring here in `getStories` is tidying up to make sure those `.stories.mdx` files don't get added to the array twice. If you are able to refactor this so we're only calling one function and not two my confusion will hopefully go away 😅 , but don't worry about it.